test/0.8.9/oracle/hashConsensus.fastLaneMembers.test.ts

```diff
@@ -0,0 +1,234 @@
+import { expect } from "chai";
+import { Signer } from "ethers";
+import { ethers } from "hardhat";
+
+import { HashConsensusTimeTravellable } from "typechain-types";
+
+import { CONSENSUS_VERSION, MAX_UINT256 } from "lib";
+
+import { deployHashConsensus, DeployHashConsensusParams, HASH_1 } from "test/deploy";
+
+const prepareFrameData = async ({
+  fastLaneMembers,
+  restMembers,
+}: {
+  fastLaneMembers: number[];
+  restMembers: number[];
+}): Promise<{ fastLaneMembers: Signer[]; restMembers: Signer[] }> => {
+  const signers = await ethers.getSigners();
+  return {
+    fastLaneMembers: fastLaneMembers.map((index) => signers[index]),
+    restMembers: restMembers.map((index) => signers[index]),
+  };
+};
+
+describe("HashConsensus:Fast-lane members", () => {
+  let admin: Signer,
+    member1: Signer,
+    member2: Signer,
+    member3: Signer,
+    member4: Signer,
+    member5: Signer,
+    stranger: Signer;
```

Comment:
```suggestion
  let admin: Signer;
  let member1: Signer;
  let member2: Signer;
  let member3: Signer;
  let member4: Signer;
  let member5: Signer;
  let stranger: Signer;
```

(just for consistency)